src/net/sourceforge/kolmafia/textui/parsetree/AggregateType.java

```diff
@@ -62,11 +62,17 @@ public AggregateType(final Type dataType, final int size) {
   }
 
   // VarArg
-  public AggregateType(
+  protected AggregateType(
       final String name, final Type dataType, final int size, final Location location) {
     this(name, dataType, DataTypes.INT_TYPE, size, false, location);
   }
 
+  // PluralValue
+  protected AggregateType(
+      final String name, final Type dataType, final Type indexType, final Location location) {
+    this(name, dataType, indexType, -1, false, location);
```

Comment:
Just as some food for thought: I wonder if the PluralValueType should actually be more like an array, by knowing its size. 
I thought it would work out of the box, but of course the PluralValue has its dataType and indexType "mixed up".